packages/verifiable-credentials/src/credentials.ts

```diff
@@ -92,6 +101,7 @@ export default class Credentials {
 	public async createVerifiablePresentation(
 		vcJwts: string[],
 		context: IContext,
+		options: CreatePresentationOptions = { removeOriginalFields: false },
 		issuer?: any,
```

Comment:
`options` should be after `issuer` for backward compatibility

Reply:
Done

Reply:
Done